ejb3/src/main/java/org/jboss/as/ejb3/remote/EJBRemoteTransactionPropagatingInterceptor.java

```diff
@@ -114,7 +114,7 @@ private void createOrResumeXidTransaction(final XidTransactionID xidTransactionI
         } else {
             // begin a new tx and add it to the tx repository
             // TODO: Fix the tx timeout to accept a value from the client (WFLY-2789)
-            final Transaction newSubOrdinateTx = this.ejbRemoteTransactionsRepository.importTransaction(xidTransactionID, Integer.MAX_VALUE);
+            final Transaction newSubOrdinateTx = this.ejbRemoteTransactionsRepository.importTransaction(xidTransactionID, Integer.parseInt(System.getProperty("remote-tx-timeout", "31536000")));
```

Comment:
https://bugzilla.redhat.com/show_bug.cgi?id=1265300#c5.
The value can be configured in the standalone configuration file of the server so that the default tx time out can be modified by the user (eg 300) like this : 
<system-properties>
        <property name="remote-tx-timeout" value="300"/>
</system-properties>
The value is not propagated, just configured in the standalone configuration file.